{{ cookiecutter.project_slug }}/api/{{ cookiecutter.project_slug }}/settings/contrib/sentry.py

```diff
@@ -7,6 +7,9 @@
 
 USE_SENTRY = env.bool("{{ cookiecutter.project_slug | upper() }}_USE_SENTRY", default=True)
 if USE_SENTRY:  # pragma: no cover
-    SENTRY_DSN = env.str("{{ cookiecutter.project_slug | upper() }}_SENTRY_DSN")
-    SENTRY_ENVIRONMENT = env.str("{{ cookiecutter.project_slug | upper() }}_SENTRY_ENVIRONMENT")
-    sentry_sdk.init(dsn=SENTRY_DSN, integrations=[DjangoIntegration()], environment=SENTRY_ENVIRONMENT)
+    sentry_kwargs = {
+        "dsn": env.str("{{ cookiecutter.project_slug | upper() }}_SENTRY_DSN"),
+        "environment": env.str("{{ cookiecutter.project_slug | upper() }}_SENTRY_ENVIRONMENT"),
+        "integrations": [DjangoIntegration()],
```

Comment:
We may add Celery integration as well.

Reply:
I just doing it on my project. Will do it as a separate PR to skeleton.